.github/workflows/release.yaml

```diff
@@ -94,3 +94,22 @@ jobs:
         if: env.SHOULD_DEPLOY_DOCS == 'true'
         id: deployment
         uses: actions/deploy-pages@v1
+
+      - name: Checkout API Docs
+        if: env.SHOULD_DEPLOY_DOCS == 'true'
+        uses: actions/checkout@v3
+        with:
+          ref: docs
+        
+      - name: Generate and Push API Docs
+        if: env.SHOULD_DEPLOY_DOCS == 'true'
+        run: |
+          pnpm install
+          pnpm build
+          rm -f apps/docs/.gitignore
+          git add apps/docs/src/api/
+          git add apps/docs/.typedoc/api-links.json
```

Comment:
Could this lead to conflicts down the road?

IIUC, the version from `master` will always contain `apps/docs/.gitignore`.

Meanwhile, we will always try to delete it, so the next sync will probably fail.

Reply:
The remove is not committed though. Just allows for the markdown and links files to be committed to `docs`. I can also `restore` the `apps/docs/.gitignore` for completeness. We could have conflicts in `docs` if we ever did start committing the contents of `apps/docs/src/api/`.